carts/carts.css

```diff
@@ -0,0 +1,54 @@
+.width-800 {
+  max-width: 950px;
+  display: flex;
+  margin: 0 auto;
+  flex-wrap: wrap;
+  gap: 1.5rem;
+}
+.cart-heading {
+  text-align: center;
+}
+.col-60-pc {
+  width: calc(60% - 2rem);
+}
+.col-40-pc {
+  width: 40%;
+  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  padding: 0.5rem 1rem;
```

Comment:
You can use CSS variiable for box-shadow

Reply:
changed to variable